code/lib/preview-api/src/modules/preview-web/docs-context/DocsContext.ts

```diff
@@ -17,7 +17,7 @@ import type { StoryStore } from '../../store';
 import type { DocsContextProps } from './DocsContextProps';
 
 export class DocsContext<TRenderer extends Renderer> implements DocsContextProps<TRenderer> {
-  private componentStoriesValue: PreparedStory<TRenderer>[];
+  private componentStoriesValue: Set<PreparedStory<TRenderer>>;
```

Comment:
My one nit seems to have dissappeared. It was that I don't understand why this needed to change to a set, that seems to unnecessarily complicate a value that is an array at the end of the day.

Reply:
The main reason was to ensure we don't add  duplicates of the same stories, but we are also ensuring that with other means, so I'm okay with reverting this.